### PR TITLE
chore: rebranding

### PR DESCRIPTION
**Pull request by swelf19**
rebranding. Ranemed org, renamed app
lidofinance -> neutron-org
gaia-wasm-zone -> neutron

### CODE REVIEW

**Review by pr0n00gler — APPROVED** [review]

File: app/app.go [hunk 1]
```diff
@@ -111,20 +111,21 @@ import (
...
-	interchaintxstypes "github.com/lidofinance/gaia-wasm-zone/x/interchaintxs/types"
+	"github.com/neutron-org/neutron/x/interchaintxs"
+	interchaintxskeeper "github.com/neutron-org/neutron/x/interchaintxs/keeper"
+	interchaintxstypes "github.com/neutron-org/neutron/x/interchaintxs/types"
 	// this line is used by starport scaffolding # stargate/app/moduleImport
 )
 
 const (
+	// should we use our own prefix name?
 	AccountAddressPrefix = "cosmos"
```
Comment (mmikev) [inline app/app.go]: neutron?

**Review by quasisamurai — APPROVED** [review]

File: app/app.go [hunk 2]
```diff
@@ -111,20 +111,19 @@ import (
...
+	"github.com/neutron-org/neutron/x/interchaintxs"
+	interchaintxskeeper "github.com/neutron-org/neutron/x/interchaintxs/keeper"
+	interchaintxstypes "github.com/neutron-org/neutron/x/interchaintxs/types"
 	// this line is used by starport scaffolding # stargate/app/moduleImport
 )
 
 const (
-	AccountAddressPrefix = "cosmos"
-	Name                 = "gaia-wasm-zoned"
+	Name                 = "neutrond"
```
Comment (ratik) [inline app/app.go:126]: so `AccountAddressPrefix` is not needed anymore?

Comment (swelf19) [reply app/app.go:126]: Actually the cons was not used anywhere in the code. Now we configure prefixes at this place https://github.com/neutron-org/neutron/blob/chore/rebranding/cmd/neutrond/main.go#L12.

**Review by NeverHappened — APPROVED** [review]

**Review by ratik — APPROVED** [review]

File: x/interchaintxs/types/tx.pb.go [hunk 3]
```diff
@@ -265,7 +264,7 @@ func NewMsgClient(cc grpc1.ClientConn) MsgClient {
 
 func (c *msgClient) RegisterInterchainAccount(ctx context.Context, in *MsgRegisterInterchainAccount, opts ...grpc.CallOption) (*MsgRegisterInterchainAccountResponse, error) {
 	out := new(MsgRegisterInterchainAccountResponse)
-	err := c.cc.Invoke(ctx, "/lidofinance.interchainadapter.interchaintxs.v1.Msg/RegisterInterchainAccount", in, out, opts...)
+	err := c.cc.Invoke(ctx, "/neutron_org.interchainadapter.interchaintxs.v1.Msg/RegisterInterchainAccount", in, out, opts...)
```
Comment (sotnikov-s) [inline x/interchaintxs/types/tx.pb.go]: don't really like this neutron **_org** suffix. let's simply use neutron?

**Review by sotnikov-s — CHANGES_REQUESTED** [review]
cmd+shift+F on `lido` still results in several matches particularly in the `./demo/` folder

**Comment by swelf19** [comment]
> cmd+shift+F on `lido` still results in several matches particularly in the `./demo/` folder

as far as i can see its only lido's contracts

File: x/interchaintxs/types/tx.pb.go (hunk 3 above)
Comment (swelf19) [reply x/interchaintxs/types/tx.pb.go]: my first thought was the same, but second thought - lidofinance its an org, so lets replace with another org. we can vote with emojis i think

**Comment by sotnikov-s** [comment]
> > cmd+shift+F on `lido` still results in several matches particularly in the `./demo/` folder
> 
> as far as i can see its only lido's contracts

do we need them in the repo? maybe we should move the helpers to the interchain staking repo? this is doubtfully the place for lido contracts: as far as I understand, it's going to become a public neutron daemon repository

File: x/interchaintxs/types/tx.pb.go (hunk 3 above)
Comment (sotnikov-s) [reply x/interchaintxs/types/tx.pb.go]: didn't get the thing about lidofinance is org, wym? we didn't have the _org suffix when the paths had lidofinance prefix

**Review by foxpy — CHANGES_REQUESTED** [review]
It doesn't look right at all.

image

**Review by NeverHappened — CHANGES_REQUESTED** [review]
Don't know if this is only my problem, but make `build && make init` fails:
```
Starting test-1 in ./data...
Creating log file at ./data/test-1.log
Starting test-2 in ./data...
Creating log file at ./data/test-2.log
Initializing relayer...
./network/hermes/restore-keys.sh
Success: Restored key 'testkey' (neutron1mjk79fjjgpplak5wq838w0yd982gzkyf8fxu8u) on chain test-1
Success: Restored key 'testkey' (neutron17dtl0mjt3t77kpuhg2edqzjpszulwhgzcdvagh) on chain test-2
./network/hermes/create-conn.sh
Initiating connection handshake...
Error: relayer error: RPC error to endpoint http://127.0.0.1:16657/: response error: Internal error: open data/test-1/data/blockstore.db/000034.sst: no such file or directory (code: -32603)
make: *** [init] Error 1
```

**Comment by swelf19** [comment]
> Don't know if this is only my problem, but make `build && make init` fails:
> 
> ```
> Starting test-1 in ./data...
> Creating log file at ./data/test-1.log
> Starting test-2 in ./data...
> Creating log file at ./data/test-2.log
> Initializing relayer...
> ./network/hermes/restore-keys.sh
> Success: Restored key 'testkey' (neutron1mjk79fjjgpplak5wq838w0yd982gzkyf8fxu8u) on chain test-1
> Success: Restored key 'testkey' (neutron17dtl0mjt3t77kpuhg2edqzjpszulwhgzcdvagh) on chain test-2
> ./network/hermes/create-conn.sh
> Initiating connection handshake...
> Error: relayer error: RPC error to endpoint http://127.0.0.1:16657/: response error: Internal error: open data/test-1/data/blockstore.db/000034.sst: no such file or directory (code: -32603)
> make: *** [init] Error 1
> ```

are there any errors above the screen? Does the error still happen? I cant reproduce it

**Comment by swelf19** [comment]
> > > cmd+shift+F on `lido` still results in several matches particularly in the `./demo/` folder
> > 
> > 
> > as far as i can see its only lido's contracts
> 
> do we need them in the repo? maybe we should move the helpers to the interchain staking repo? this is doubtfully the place for lido contracts: as far as I understand, it's going to become a public neutron daemon repository

yeah, lets remove demo contracts from this repo. I think we can create a standalone repo with general purpose contracts for tests in nearest future

**Comment by NeverHappened** [comment]
> https://github.com/neutron-org/neutron/pull/15#issuecomment-1178988500

I've added the neutrond to path and it fixed the problem, sry for bothering

**Review by foxpy — APPROVED** [review]

File: readme.md [hunk 4]
```diff
@@ -50,51 +50,51 @@ ignite generate proto-go
...
+1. Start 2 neutron-chains and an IBC relayer:
 ```
 make init && make start-rly
 ```
 
 ### terminal 2
 1. Register an interchain query to get delegations of delegator on remote chain:
 ```
-gaia-wasm-zoned tx interchainqueries register-interchain-query test-2 connection-0 x/staking/DelegatorDelegations '{"delegator": "cosmos1qnk2n4nlkpw9xfqntladh74w6ujtulwn7j8za9"}' 1 --from demowallet1 --gas 10000000 --gas-adjustment 1.4 --gas-prices 0.5stake --broadcast-mode block --chain-id test-1 --keyring-backend test --home ./data/test-1 --node tcp://127.0.0.1:16657
+neutrond tx interchainqueries register-interchain-query test-2 connection-0 x/staking/DelegatorDelegations '{"delegator": "cosmos1qnk2n4nlkpw9xfqntladh74w6ujtulwn7j8za9"}' 1 --from demowallet1 --gas 10000000 --gas-adjustment 1.4 --gas-prices 0.5stake --broadcast-mode block --chain-id test-1 --keyring-backend test --home ./data/test-1 --node tcp://127.0.0.1:16657
```
Comment (mmikev) [inline readme.md]: Still old cosmos prefixed addresses in readme

**Review by pr0n00gler — CHANGES_REQUESTED** [review]

File: readme.md (hunk 4 above)
Comment (foxpy) [reply readme.md]: Actually, `grep -R cosmos1 .` reveals lots of them.

**Review by pr0n00gler — APPROVED** [review]

**Review by ratik — APPROVED** [review]

**Review by sotnikov-s — APPROVED** [review]

**Review by NeverHappened — APPROVED** [review]

**Pull request merged**